packages/guides-restructured-text/src/RestructuredText/DependencyInjection/ReStructuredTextExtension.php

```diff
@@ -21,13 +25,34 @@
 use function strrchr;
 use function substr;
 
-class ReStructuredTextExtension extends Extension implements PrependExtensionInterface, CompilerPassInterface
+class ReStructuredTextExtension extends Extension implements PrependExtensionInterface, CompilerPassInterface, ConfigurationInterface
 {
     private const HTML = [VersionChangeNode::class => 'body/version-change.html.twig'];
 
+    public function getConfigTreeBuilder(): TreeBuilder
+    {
+        $treeBuilder = new TreeBuilder('restructured_text');
+        $rootNode = $treeBuilder->getRootNode();
+        assert($rootNode instanceof ArrayNodeDefinition);
+
+        $rootNode
+            ->children()
+                ->scalarNode('default_code_language')->defaultValue('')->end()
```

Comment:
Once this is merged, I suppose #475 should use that parameter?

Reply:
No, it's works automatically: `CodeNode`'s have a `language` property. If it's not explicitly set (e.g. by the `.. code-block:: html` directive), it'll default to this config property.

The highlighter can just use `codeNode.language`.